src/services/authenticator.js

```diff
@@ -51,29 +50,17 @@ function Authenticator() {
     || 'Invalid token. Please enter your authentication token.';
 
   this.logout = async (opts = {}) => {
-    const path = `${os.homedir()}/.forestrc`;
-
-    return new P((resolve, reject) => {
-      fs.stat(path, (err) => {
-        if (err === null) {
-          fs.unlinkSync(path);
-
-          if (opts.log) {
-            console.log(chalk.green('👍  You\'re now unlogged 👍 '));
-          }
-
-          resolve();
-        } else if (err.code === 'ENOENT') {
-          if (opts.log) {
-            logger.error('You are not logged');
-          }
-
-          resolve();
-        } else {
-          reject(err);
-        }
-      });
-    });
+    const forestrcPath = `${os.homedir()}/.forestrc`;
+    const isForestLoggedIn = this.getVerifiedToken(forestrcPath);
+    const lumbercPath = `${os.homedir()}/.lumberrc`;
+    const isLumberLoggedIn = this.getVerifiedToken(lumbercPath);
+
+    if (isForestLoggedIn) {
+      fs.unlinkSync(forestrcPath);
+    }
+    if (opts.log) {
+      logger.info(`You are unlogged. ${isLumberLoggedIn ? '(still connected via lumber)' : ''}`);
```

Comment:
"(still connected via lumber)" => so Forest CLI will behave as if you were logged.
if really unlogged:
`console.log(chalk.green('You are now unlogged.'));`
(Why do you remove the color by the way? info is green?)

if logged in with lumber you should display a very specific message:
"You cannot be logged out with this command. Please use lumber logout" (something like that).

Reply:
about green color: yes, logger is preferred over console, it manage the console. I've set it to green, let me know if it's not better.